suleyunus/quick-credit
# #165875615 Set up ESLint and configure style guide

#### What does this PR do?

Sets up ESLint and configures project to follow airbnb style guide

#### Description of Task to be completed?

Set up linting library and ensure work follows specified style guide

#### How should this be manually tested?

After cloning repo, cd into it and create testing.js file in your editor of choice and in the js file write:

```javascript
var num = 5;
```

then in your command line under the directory type

```
npm run eslint testing.js
```
you should see errors that indicate the file is not following specified style guide(airbnb)

#### What are the relevant pivotal tracker stories?

#165875615
